.travis/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [ ! -z "$TRAVIS_TAG" ]
+then
+    echo "on a tag -> set pom.xml <version> to $TRAVIS_TAG"
```

Comment:
Will it trigger on any tag? Wouldn’t it be better to only do it for tags like ‘vX.Y.Z’. It would require more scripting but I think it will avoid accidental releases. What do you think?

Reply:
You are right, but i can solve this in the  travis config.

Reply:
Can you? if the tag starts with a 'v' I think something can break, can't it?

Reply:
To be more constructive, here is something you can do:
```
if ( echo $TRAVIS_TAG | egrep -i '^v\d+\.\d+\.\d+$' )
then
  VERSION_NUMBER=${TRAVIS_TAG:1} # Removes the initial 'v' character
   # continue using $VERSION_NUMBER  instead of $TRAVIS_TAG
```

Reply:
```xml
 on:
      all_branches: true
      tags: true
      jdk: oraclejdk8
      condition: $TRAVIS_TAG =~ ^v[0-9]+\.[0-9]+\.[0-9]+
```

Reply:
if the tag is empty, then no release are triggered.

Reply:
Yes, but you still need to remove the initial "v" using something like the code I posted above